t/base/rs.t

```diff
@@ -1,312 +1,455 @@
 open TESTFILE, "<./foo";
 binmode TESTFILE;
 test_string(*TESTFILE);
 close TESTFILE;
+
 unlink "./foo";
 
-# try the record reading tests. New file so we don't have to worry about
+# ----------------------------------------------------------------------
+# Try the record reading tests. New file so we don't have to worry about
 # the size of \n.
+# ----------------------------------------------------------------------
 open TESTFILE, ">./foo";
 print TESTFILE $teststring2;
 binmode TESTFILE;
 close TESTFILE;
+
 open TESTFILE, "<./foo";
 binmode TESTFILE;
 test_record(*TESTFILE);
 close TESTFILE;
 $test_count_end = $test_count;  # Needed to know how many tests to skip
 
+# -----------------------------------
+# $/ preserved when set to bad value
+# -----------------------------------
 $/ = "\n";
-my $note = "\$/ preserved when set to bad value";
+
 # none of the setting of $/ to bad values should modify its value
 test_bad_setting();
+
 print +($/ ne "\n" ? "not " : "") .
   "ok $test_count # \$/ preserved when set to bad value\n";
 ++$test_count;
+# -----------------------------------
+
+# =================================
+# VMS case
+# =================================
 
 # Now for the tricky bit--full record reading
-if ($^O eq 'VMS') {
-  # Create a temp file. We jump through these hoops 'cause CREATE really
-  # doesn't like our methods for some reason.
-  open FDLFILE, "> ./foo.fdl";
-  print FDLFILE "RECORD\n FORMAT VARIABLE\n";
-  close FDLFILE;
-  open CREATEFILE, "> ./foo.com";
-  print CREATEFILE '$ DEFINE/USER SYS$INPUT NL:', "\n";
-  print CREATEFILE '$ DEFINE/USER SYS$OUTPUT NL:', "\n";
-  print CREATEFILE '$ OPEN YOW []FOO.BAR/WRITE', "\n";
-  print CREATEFILE '$ CLOSE YOW', "\n";
-  print CREATEFILE "\$EXIT\n";
-  close CREATEFILE;
-  $throwaway = `\@\[\]foo`, "\n";
-  open(TEMPFILE, ">./foo.bar") or print "# open failed $! $^E\n";
-  print TEMPFILE "foo\nfoobar\nbaz\n";
-  close TEMPFILE;
-
-  open TESTFILE, "<./foo.bar";
-  $/ = \10;
-  $bar = <TESTFILE>;
-  if ($bar eq "foo\n") {print "ok $test_count\n";} else {print "not ok $test_count\n";}
-  $test_count++;
-  $bar = <TESTFILE>;
-  if ($bar eq "foobar\n") {print "ok $test_count\n";} else {print "not ok $test_count\n";}
-  $test_count++;
-  # can we do a short read?
-  $/ = \2;
-  $bar = <TESTFILE>;
-  if ($bar eq "ba") {print "ok $test_count\n";} else {print "not ok $test_count\n";}
-  $test_count++;
-  # do we get the rest of the record?
-  $bar = <TESTFILE>;
-  if ($bar eq "z\n") {print "ok $test_count\n";} else {print "not ok $test_count\n";}
-  $test_count++;
-
-  close TESTFILE;
-  1 while unlink qw(foo.bar foo.com foo.fdl);
-} else {
-  # Nobody else does this at the moment (well, maybe OS/390, but they can
-  # put their own tests in) so we just punt
-  foreach $test ($test_count..$test_count + 3) {
-      print "ok $test # skipped on non-VMS system\n";
-      $test_count++;
-  }
+if ( $^O eq 'VMS' ) {
```

Comment:
reformatting large parts of a file for whitespace-only changes means that 'git blame' now reports the wrong commit, which causes more work for bisection and other archaeological endeavours.

Reply:
In perlhack.pod there are such lines:

> Use spaces for indentation, not tab characters.
> 
> The codebase is a mixture of tabs and spaces for indentation, and we
> are moving to spaces only.  Converting lines you're patching from 8-wide
> tabs to spaces will help this migration.

That means that potentially there are a lot of **similar places** in perl core where should be done such reformatting "which cases more work for bisection and other archaeological endeavours". It cannot be avoided it we want to have a modern (not slovenly) perl code in the repo.